README.md

```diff
@@ -37,18 +44,38 @@ Instructions on how to use them are provided below.
 | ------ | ------ |
 | help | Shows all available commands |
 | hello | Greeting command |
-| add-contact | Adds new contact. Required arguments: name, phone (10 digits) |
+| add-contact | Adds new contact. Required arguments: name, phone (10 digits). Optional argument: email, birthday |
```

Comment:
we don't have arguments for add-contact command, but we have subset of questions to fulfil. Just like in add-note command